src/sensor.cc

```diff
@@ -0,0 +1,82 @@
+#include "sensor.h"
+#include "vector3.h"
```

Comment:
You should include `vector3-inl.h` to instead of `vector3.h`, or it can't find the template implement symbol:

```
dyld[3143]: missing symbol called
[1]    3143 abort      node example/sensor.js
```

```
(lldb) bt
* thread #1, queue = 'com.apple.main-thread', stop reason = signal SIGABRT
  * frame #0: 0x0000000104d0c0ce dyld`__abort_with_payload + 10
    frame #1: 0x0000000104d221cf dyld`abort_with_payload_wrapper_internal + 80
    frame #2: 0x0000000104d22201 dyld`abort_with_payload + 9
    frame #3: 0x0000000104cd0613 dyld`dyld4::halt(char const*) + 375
    frame #4: 0x0000000104cecc6e dyld`dyld4::APIs::_dyld_missing_symbol_abort() + 34
    frame #5: 0x0000000105ae8138 binding.node`node_sfml::sensor::GetValue(info=0x00007ff7bfefe578) at sensor.cc:73:16 [opt]
    frame #6: 0x0000000105ad70ad binding.node`Nan::imp::FunctionCallbackWrapper(info=0x00007ff7bfefe5f0) at nan_callbacks_12_inl.h:176:3 [opt]
    frame #7: 0x00000001002b6dd9 node`v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) + 265
    frame #8: 0x00000001002b68a6 node`v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) + 550
    frame #9: 0x00000001002b601f node`v8::internal::Builtin_HandleApiCall(int, unsigned long*, v8::internal::Isolate*) + 255
    frame #10: 0x0000000100b26eb9 node`Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_BuiltinExit + 57
    frame #11: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #12: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #13: 0x0000000100b43424 node`Builtins_ArrayMap + 1284
    frame #14: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #15: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #16: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #17: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #18: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #19: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #20: 0x0000000100ab98aa node`Builtins_InterpreterEntryTrampoline + 202
    frame #21: 0x0000000100ab7ab8 node`Builtins_JSEntryTrampoline + 88
    frame #22: 0x0000000100ab7843 node`Builtins_JSEntry + 131
    frame #23: 0x000000010036ece8 node`v8::internal::(anonymous namespace)::Invoke(v8::internal::Isolate*, v8::internal::(anonymous namespace)::InvokeParams const&) + 2520
    frame #24: 0x000000010036e2f5 node`v8::internal::Execution::Call(v8::internal::Isolate*, v8::internal::Handle<v8::internal::Object>, v8::internal::Handle<v8::internal::Object>, int, v8::internal::Handle<v8::internal::Object>*) + 213
    frame #25: 0x0000000100269fb1 node`v8::Function::Call(v8::Local<v8::Context>, v8::Local<v8::Value>, int, v8::Local<v8::Value>*) + 465
    frame #26: 0x0000000100096e5e node`node::ExecuteBootstrapper(node::Environment*, char const*, std::__1::vector<v8::Local<v8::String>, std::__1::allocator<v8::Local<v8::String> > >*, std::__1::vector<v8::Local<v8::Value>, std::__1::allocator<v8::Local<v8::Value> > >*) + 110
    frame #27: 0x0000000100097f56 node`node::StartExecution(node::Environment*, char const*) + 374
    frame #28: 0x0000000100097c99 node`node::StartExecution(node::Environment*, std::__1::function<v8::MaybeLocal<v8::Value> (node::StartExecutionCallbackInfo const&)>) + 809
    frame #29: 0x00000001000064fd node`node::LoadEnvironment(node::Environment*, std::__1::function<v8::MaybeLocal<v8::Value> (node::StartExecutionCallbackInfo const&)>) + 109
    frame #30: 0x000000010010fefd node`node::NodeMainInstance::Run(int*, node::Environment*) + 61
    frame #31: 0x000000010010fb79 node`node::NodeMainInstance::Run(node::EnvSerializeInfo const*) + 121
    frame #32: 0x0000000100099f68 node`node::Start(int, char**) + 200
    frame #33: 0x0000000104ccb4fe dyld`start + 462
```